lang/vi_vn.lang

```diff
@@ -0,0 +1,954 @@
+option.FAR_VANILLA_FOG.comment=Kích hoạt sương mù giống như vanilla. §e[*]§rTùy chọn này bị ảnh hưởng bởi mật độ sương mù.
+value.FAR_VANILLA_FOG.0=§cTẮT
+value.FAR_VANILLA_FOG.1=Chỉ Thế giới trên mặt đất
+value.FAR_VANILLA_FOG.2=Chỉ Nether & Kết thúc
+value.FAR_VANILLA_FOG.3=Toàn bộ
+
+option.FAR_VANILLA_FOG_STYLE=Phong cách Sương mù Vanilla Xa
+option.FAR_VANILLA_FOG_STYLE.comment=Spherical hoạt động giống như sương mù vanilla. Cylindrical bỏ qua trục Y.
+value.FAR_VANILLA_FOG_STYLE.0=Hình cầu
+value.FAR_VANILLA_FOG_STYLE.1=Hình trụ
+
+option.FOG_DENSITY_VANILLA=Mật độ Vanilla
+option.FOG_DENSITY_VANILLA.comment=Điều chỉnh mật độ cho sương mù vanilla.
+
+option.FOG_VANILLA_CLOUD=Sương mù Mây Vanilla
+option.FOG_VANILLA_CLOUD.comment=Kích hoạt sương mù trên mây vanilla.
+value.FOG_VANILLA_CLOUD.0=§cTẮT
+value.FOG_VANILLA_CLOUD.1=Tối thiểu
+value.FOG_VANILLA_CLOUD.2=Giảm bớt
+value.FOG_VANILLA_CLOUD.3=Toàn bộ
+
+#Cấu hình Hộp trời
+option.ROUND_SUN_MOON=Mặt trời & Mặt trăng Tròn
+option.ROUND_SUN_MOON.comment=Kích hoạt mặt trời & mặt trăng dựa trên shader. §e[*]§rTắt mặt trời & mặt trăng vanilla trong Cài đặt Video > Chi tiết > Mặt trời & Mặt trăng trên 1.16.4 hoặc thấp hơn.
+
+option.ROUND_SUN_MOON_SIZE=Kích thước Mặt trời & Mặt trăng
+option.ROUND_SUN_MOON_SIZE.comment=Điều chỉnh kích thước cho mặt trời & mặt trăng tròn và điểm sáng đặc biệt.
+
+option.STARS=Sao
+option.STARS.comment=Kích hoạt sao.
+
+option.AURORA=Aurora
+option.AURORA.comment=Kích hoạt aurora. §a[+]§rHiệu ứng này được hưởng lợi từ TAA.
+
+option.SKYBOX_INTENSITY=Cường độ Hộp trời
+option.SKYBOX_INTENSITY.comment=Điều chỉnh cường độ hộp trời.
+
+option.SKYBOX_OPACITY=Độ mờ Hộp trời
+option.SKYBOX_OPACITY.comment=Điều chỉnh độ mờ hộp trời. §c[-]§rChỉ có sẵn cho 1.16.5 trở lên.
+
+option.SUN_INTENSITY=Cường độ Mặt trời
+option.SUN_INTENSITY.comment=Điều chỉnh cường độ mặt trời hộp trời. §c[-]§rChỉ có sẵn cho 1.16.5 trở lên.
+
+option.MOON_INTENSITY=Cường độ Mặt trăng
+option.MOON_INTENSITY.comment=Điều chỉnh cường độ mặt trăng hộp trời. §c[-]§rChỉ có sẵn cho 1.16.5 trở lên.
+
+#Nước
+option.WATER_MODE=Chế độ Texture Nước
+option.WATER_MODE.comment=Xác định liệu texture nước vanilla có nên được sử dụng hay không. Mặc định: màu nước phẳng của shader, không có texture. Vanilla: texture vanilla. Vanilla Được Tô màu: texture vanilla với màu nước của shader. Vanilla Phẳng: màu nước phẳng của vanilla.
+value.WATER_MODE.0=Mặc định
+value.WATER_MODE.1=Vanilla
+value.WATER_MODE.2=Vanilla Được Tô màu
+value.WATER_MODE.3=Vanilla Phẳng
+
+option.WATER_ALPHA_MODE=Chế độ Alpha Nước
+option.WATER_ALPHA_MODE.comment=Xác định liệu nước sử dụng giá trị alpha dựa trên shader hay texture. §e[*]§rMột số hiệu ứng sẽ tiếp tục sử dụng giá trị alpha dựa trên shader.
+value.WATER_ALPHA_MODE.0=Phẳng
+value.WATER_ALPHA_MODE.1=Vanilla
+
+option.WATER_NORMALS=Bản đồ Bình thường Nước
+option.WATER_NORMALS.comment=Kích hoạGPT
+t ánh xạ bình thường được tạo trong thời gian thực cho nước.
```

Comment:
please write setting comment/description in one line

Reply:
I fix done